pkg/deploy/api/v1/types.go

```diff
@@ -384,6 +384,20 @@ type DeploymentConfigRollbackSpec struct {
 	IncludeStrategy bool `json:"includeStrategy"`
 }
 
+// DeploymentRequest is a request to a deployment config for a new deployment.
+type DeploymentRequest struct {
+	unversioned.TypeMeta `json:",inline"`
+	// Name of the deployment config for requesting a new deployment.
+	Name string `json:"name"`
+	// If true, the updated deployment config will not be persisted in storage.
+	DryRun bool `json:"dryRun,omitempty"`
+	// Latest will update the deployment config with the latest images found in the cluster.
+	Latest bool `json:"latest"`
+	// Force a new deployment independently of any triggers eg. this will override non-automatic
+	// image change triggers.
+	Force bool `json:"force"`
```

Comment:
Seems like this should be `omitempty`.  Should cleanly default to false and most people won't set it, right?